play-json/src/main/scala/org/http4s/play/instances/package.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package play
+
+package object instances {
+  object all extends AllInstances
+  object play extends PlayInstances
```

Comment:
There is syntax in this: the `MessageSyntax`.  That would typically go in a `syntax` object in the Cats encoding.  Interestingly, it depends on `jsonOf`, so we need to think about where that belongs.  I've called it out on Gitter to see what the group thinks.